plugins/source/cloudflare/client/client.go

```diff
@@ -124,7 +124,6 @@ func Configure(ctx context.Context, logger zerolog.Logger, spec *Spec) (schema.C
 		if err != nil {
 			return nil, err
 		}
-		c.AccountID = account.AccountId
```

Comment:
What's the reason this is removed?

Reply:
This was removed because `AccountID` is no longer a property on the `cloudflare.API` struct as of `0.70.0`. https://github.com/cloudflare/cloudflare-go/commit/385fc64122fe2d8af5a5f3b4b6a08aa87c5dcca8. I searched for additional references in this plugin and found none.

Reply:
Ah got it. I got confused by the usage of our own `client.AccountID`